src/System.Diagnostics.Process/src/System/Diagnostics/Process.OSX.cs

```diff
@@ -75,8 +73,12 @@ private unsafe IntPtr ProcessorAffinityCore
         /// </summary>
         private void GetWorkingSetLimits(out IntPtr minWorkingSet, out IntPtr maxWorkingSet)
         {
-            // TODO: Implement this
-            throw NotImplemented.ByDesign;
+            // Minimum working set (or resident set, as it is called on *nix) doesn't exist so set to 0
+            minWorkingSet = IntPtr.Zero;
+
+            // Get the max working set size
+            Interop.libc.rlimit info = Interop.libc.getrlimit(Interop.libc.RLIMIT_Resources.RLIMIT_RSS);
```

Comment:
Good catch. I couldn't find a way to set this on other processes so I'll throw a PlatformNotSupported exception in this scenario